flexget/components/sites/sites/newznab.py

```diff
@@ -102,14 +106,10 @@ def do_search_tvsearch(self, arg_entry, task, config=None):
         ):
             return []
         if arg_entry.get('tvrage_id'):
-            lookup = '&rid=%s' % arg_entry.get('tvrage_id')
+            lookup = f"&rid={arg_entry.get('tvrage_id')}"
         else:
-            lookup = '&q=%s' % quote(arg_entry['series_name'])
-        url = (
-            config['url']
-            + lookup
-            + '&season=%s&ep=%s' % (arg_entry['series_season'], arg_entry['series_episode'])
-        )
+            lookup = f"&q={quote(arg_entry['series_name'])}"
+        url = f"{config['url']}{lookup}&season={arg_entry['series_season']}&ep={arg_entry['series_episode']}"
```

Comment:
i think that if you use the params keyword, you could avoid using the urlquote as it does so internally via requests:
```python
params = {
    'season': arg_entry['series_season'],
    'ep': arg_entry['series_episode'],
    'q': arg_entry['series_name'],
}
```

Reply:
I think I formatted everything correctly. Found another place where we needed to make sure 'title' was encoded for URLs, too. Removed import, and changed one line to use f-strings.